inst/tutorials/chapter4/lm_example.Rmd

```diff
@@ -134,8 +134,9 @@ surf <- acast(df, avginc ~ str)
 
 Caschool %>%
   plot_ly() %>%
-  add_markers(x = ~str, y = ~avginc, z = ~testscr, name = "Data", hoverinfo = "skip", opacity = .75, marker=list(color = 'green', size = 5)) %>%
-  add_surface(x = to_plot_x, y = to_plot_y, z = ~surf, inherit = F, name = "Best Fit Plane", opacity = .75) %>%
+  add_markers(x = ~str, y = ~avginc, z = ~testscr, name = "Data", hoverinfo = "skip", opacity = .75, marker=list(color = 'red', size = 3)) %>%
+  add_surface(x = to_plot_x, y = to_plot_y, z = ~surf, inherit = F, name = "Best Fit Plane", opacity = .75, cauto = F, surfacecolor = rep('grey', 4)) %>%
+  hide_colorbar() %>%
```

Comment:
much better! do you like the red points? not a strong opinion on the color.
* by the way: i would remove the title of this plot. because if you touch the plot you see the menu bar, and it overalys the title, and it looks bad.